components/PopupAssignee.vue

```diff
@@ -1,50 +1,91 @@
+          <template v-slot:activator="{ props }">
+            <v-icon icon="mdi-account-plus-outline" v-bind="props" />
+          </template>
+        </v-tooltip>
+
+        <v-tooltip v-else :text="assignee.displayName">
+          <template v-slot:activator="{ props }">
+            <v-avatar v-if="assignee.avatar" v-bind="props">
+              <v-img :src="assignee.avatar" />
+            </v-avatar>
+            <v-avatar v-else color="indigo" v-bind="props">
+              {{ assignee.initials }}
+            </v-avatar>
+          </template>
+        </v-tooltip>
+      </v-btn>
     </template>
-  </v-autocomplete>
+
+    <v-card class="card">
+      <v-text-field v-model="state.search" label="Search" hide-details autofocus @blur="state.search = ''"/>
+      <v-list class="autocomplete__list">
+        <v-list-item
+          v-for="user in filteredUsers"
+          :title="user.displayName"
+          class="autocomplete__list-item"
+          @click="() => handleNewAssigneeSelect(user)"
+        >
+          <template v-slot:prepend="{}">
+            <v-avatar
+              v-if="user.avatar"
+              v-bind="props"
+              size="28"
+              class="avatar"
+            >
+              <v-img :src="user.avatar" />
+            </v-avatar>
+            <v-avatar
+              v-else
+              v-bind="props"
+              size="28"
+              class="avatar"
+              color="indigo"
+            >
+              {{ user.initials }}
+            </v-avatar>
+          </template>
+        </v-list-item>
+      </v-list>
+    </v-card>
+  </v-menu>
 </template>
 
 <script lang="ts" setup>
-import { ref } from 'vue'
-import { useUsersStore } from '~/store/users'
-const props = defineProps(['assigneeId', 'readOnly'])
-const userStore = useUsersStore()
-const assignee = props.assigneeId ? userStore.getUserById(props.assigneeId) : null
-const localAssignee = ref(assignee)
-const users = userStore.all().map(user => {
-  const name = `${user.firstName} ${user.lastName}`
-  return {
-    id: user.id,
-    name,
-    avatar: user.avatarUrl,
-    initials: name.trim().toUpperCase().split(' ', 2).map(str => str.charAt(0)).join('')
-  }
+import { ref } from "vue";
+import { useUsersStore } from "~/store/users";
+import { User } from "~/models/User";
+
+const props = defineProps(["assigneeId", "readOnly"]);
+const emit = defineEmits(['selectAssignee'])
+const userStore = useUsersStore();
+
+const users = userStore.all();
+const assignee = computed(() => userStore.getUserById(props.assigneeId));
+const state = reactive({
+  search: "",
+  isMenuOpen: false
 })
+const filteredUsers = computed(() => fuzzySearch(users, state.search))
+
+function fuzzySearch(users: User[] , searchText: string) {
+  if (!searchText) {      // No filter on user if searchText is empty
+    return users;
+  }
+
+  const searchRegex = new RegExp(searchText.split('').join('.*?'), 'i');
```

Comment:
please add explanation for this

Reply:
Will do so in a different PR. This code is not part of this PR as the original PR has been merged.

Reply:
Should now not show up in this PR